iwyu_tool.py

```diff
@@ -123,6 +185,8 @@ def custom_help():
 
     parser.add_argument('-v', '--verbose', action='store_true',
                         help='Print IWYU commands')
+    parser.add_argument('-c', '--clang-style', action='store_true',
```

Comment:
I wonder if you could make this a `--output` argument instead, with a set of fixed options, e.g. `iwyu`, `clang`, etc. That way it's more extensible. I'm already dreaming of plain diff output...

Something like:

    parser.add_argument('-o', '--output', type=str,
                        choices=['iwyu', 'clang'], default='iwyu',
                        help='Output format')